README.md

```diff
@@ -62,48 +62,85 @@ instance HasDefinitions MyContracts where
-          , pcPort = 9080
-          , pcProtocolParams = protocolParams
-          , pcTipPollingInterval = 10_000_000
-          , pcOwnPubKeyHash = "0f45aaf1b2959db6e5ff94dbb1f823bf257680c3c723ac2d49f97546"
-          , pcOwnStakePubKeyHash = Nothing
-          , -- Directory name of the script and data files
-            pcScriptFileDir = "./scripts"
-          , -- Directory for the signing key file(s)
-            pcSigningKeyFileDir = "./signing-keys"
-          , -- Directory where the encoded transaction files will be saved
-            pcTxFileDir = "./txs"
-          , -- Dry run mode will build the tx, but skip the submit step
-            pcDryRun = False
-          , pcLogLevel = Debug
-          , -- Protocol params file location relative to the cardano-cli working directory (needed for the cli)
-          , pcProtocolParamsFile = "./protocol.json"
-          , pcEnableTxEndpoint = True
-          -- Save some stats during contract run (only transactions execution budgets supported atm)
-          , pcCollectStats = False
-          }
+  pabConf <-
+    either error id
+      <$> BotPlutusInterface.loadPABConfig "./pabConfig.value"
   BotPlutusInterface.runPAB @MyContracts pabConf
 ```
 
+Configuration format (example: <examples/plutus-game/pabConfig.value>): 
+
+``` console
+$ cabal repl --disable-optimisation --repl-options -Wwarn
+...
+BotPlutusInterface> :m Prelude
+...
+Prelude> :l BotPlutusInterface.Config
+...
+Prelude BotPlutusInterface.Config> putStrLn docPABConfig 
+Top-level configuration file fields:
+    cliLocation: `local` or destination text
+        calling the cli through ssh when set to destination (default:
+        local)
+    chainIndexUrl: url text
+         (default: "http://localhost:9083")
+    networkId: `mainnet` or 32-bit unsigned integral number
+         (default: 42)
+    slotConfig: SlotConfig configuration
+         (see default in example)
+    scriptFileDir: path text
+        Directory name of the script and data files (default:
+        "./result-scripts")
+    signingKeyFileDir: path text
+        Directory name of the signing key files (default: "./signing-keys")
+    txFileDir: path text
+        Directory name of the transaction files (default: "./txs")
+    protocolParamsFile: filepath text
+        Protocol params file location relative to the cardano-cli working
+        directory (needed for the cli) in JSON format.  (default:
+        "./protocol.json")
+    dryRun: `true` or `false`
+        Dry run mode will build the tx, but skip the submit step (default:
+        true)
+    logLevel: `error` or `warn` or `notice` or `info` or `debug`
+         (default: info)
+    ownPubKeyHash: PubKeyHash text
+         (default: "")
+    ownStakePubKeyHash: `nothing` or StakePubKeyHash text
+         (default: nothing)
+    tipPollingInterval: non-negative integral number
+         (default: 10000000)
+    forceBudget: `nothing` or ExecutionUnits configuration
```

Comment:
This example seems to be outdated, as there is no force budget setting anymore in `PABConfig`.
And `collectStats` not presented.